web/lib/rpi-auth.js

```diff
@@ -42,11 +43,18 @@ function getLogoutRedirectUri() {
 
 function getRegisterRedirectUri() {
   const params = new URLSearchParams();
-  params.set('brand', 'coderdojo');
+  params.set('brand', brand);
   params.set('returnTo', `${homeServer}${loginPath}`);
   return `${profileServer}${profileSignupPath}?${params}`;
 }
 
+function getEditRedirectUri() {
+  const params = new URLSearchParams();
+  params.set('brand', brand);
```

Comment:
Same as above.

Reply:
Again up on L18, I'm gonna clean up that block of variables up there, I've left 'em very messy!